### PR TITLE
Heroku deployment

### PR DESCRIPTION
**Pull request by Shehryar21**

### CODE REVIEW

File: config/getter.go [hunk 1]
```diff
@@ -81,6 +81,15 @@ func GetProfilerPath() string {
 
 // -- server --
 
+func GetPort() *int {
+	if !viper.IsSet("port") {
```
Comment (AbhiPrasad) [inline config/getter.go:85]: Can’t we set a default port in code and return an int instead of a pointer?

**Review by dhillondeep — APPROVED** [review]

**Review by AbhiPrasad — COMMENTED** [review]
Golang needs actual tagged union types ngl.

Imagine golang with [Rust Optionals](https://doc.rust-lang.org/rust-by-example/std/option.html) or [Haskell Maybe](https://hackage.haskell.org/package/base-4.14.0.0/docs/Data-Maybe.html)

File: config/getter.go (hunk 1 above)
Comment (dhillondeep) [reply config/getter.go:85]: yeah but it will be hard for us to tell if $PORT env is provided to us

**Comment by dhillondeep** [comment]
@AbhiPrasad lol sorry I merged the code didn't know you were still reviewing

File: config/getter.go (hunk 1 above)
Comment (AbhiPrasad) [reply config/getter.go:85]: Ah word, that’s a good point.

**Pull request merged**